### ISSUE TITLE
Regression: location header is made absolute with redirect=manual

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-fetch#131]
**Description**
fetching a redirect response manually should not alter the location header. but since v2, it is made absolute.

eg:

```
$ curl -sD - 'https://httpbingo.org/redirect-to?url=/foo.html'
HTTP/2 302
access-control-allow-credentials: true
access-control-allow-origin: *
location: /foo.html
date: Thu, 04 Feb 2021 03:10:22 GMT
content-length: 0
server: Fly/7226d36 (2021-01-25)
via: 2 fly.io

```

but with fetch:

```js
    it('supports redirect: manual with path location', async () => {
      const resp = await fetch(`${protocol}://httpbingo.org/redirect-to?url=/foo.html&status_code=307`, { redirect: 'manual', cache: 'no-store' });
      assert.strictEqual(resp.status, 307);
      assert.strictEqual(resp.headers.get('location'), '/foo.html');
      assert.strictEqual(resp.redirected, false);
    });
```

```
AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
+ actual - expected

+ 'http://httpbingo.org/foo.html'
- '/foo.html'
Expected :/foo.html
Actual   :http://httpbingo.org/foo.html
```

**To Reproduce**
see failing test in: https://github.com/adobe/helix-fetch/pull/130

**Comment by stefan-guggisberg** [comment]
It's not a bug but actually a feature in `node-fetch` which `helix-fetch` supports (to ease migration from `node-fetch` to `helix-fetch`): 

https://github.com/node-fetch/node-fetch/blob/master/src/index.js#L116-L120

If there's consensus that we don't want it I'll remove it.

**Comment by tripodsan** [comment]
> to ease migration from node-fetch to helix-fetch

but we don't need that :-) we migrate from helix-fetch v1 to v2.. and in v1, the location header wasn't updated.

> If there's consensus that we don't want it I'll remove it.

yes, I would remove it. I think the client should decide what to do. 

(btw, for proxy implementations, this is a problem, and we would need to remove the host manually, see:  https://github.com/adobe/helix-dispatch/blob/79bb1e8083318152e959eb0312c2596e46e78acb/src/redirects.js#L49-L55)

@trieloff  WDYT?

**Comment by stefan-guggisberg** [comment]
fixed in ae78c75
